ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: 'Make sure {{ cifmw_set_openstack_containers_basedir }} exists'
+  ansible.builtin.file:
+    path: "{{ cifmw_set_openstack_containers_basedir }}/artifacts"
+    state: directory
+
+- name: Set container tag from dlrn md5 hash file
+  when:
+    - cifmw_set_openstack_containers_tag_from_md5 | bool
+  block:
+    - name: Get the md5 hash from hash file
+      ansible.builtin.slurp:
+        src: "{{ cifmw_set_openstack_containers_dlrn_md5_path }}"
+      register: dlrn_md5
+
+    - name: Set container tag
+      ansible.builtin.set_fact:
+        cifmw_set_openstack_containers_tag: "{{ dlrn_md5['content'] | b64decode | trim }}"
+        cacheable: true
+
+- name: Set the operator name
+  ansible.builtin.set_fact:
+    operator_name: "{{ cifmw_set_openstack_containers_operator_name }}"
+    cacheable: true
+
+- name: Get operator name
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: oc get pods -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o name | grep {{ operator_name }}-operator-controller-manager
```

Comment:
this seems to be the first command calling `oc` and it fails in molecule.

Apparently it's linked to the `cifmw_path` not properly set in the molecule env?
`The field 'environment' has an invalid value, which includes an undefined variable. The error was: 'cifmw_path' is undefined. 'cifmw_path' is undefined`
https://logserver.rdoproject.org/19/319/8e2c3ff4b8cd67db18c02ff533f450bc00a46c61/github-check/cifmw-molecule-set_openstack_containers/486eb38/report.html

Reply:
Alsooo... maybe we want to NOT `| grep` since it would mask potential errors. Maybe better to get the `oc` output in a `register`, and parse it? Using `| map('regexp',  operator_name~'operator-controller-manager')` or something like that, we should be able to get the actual value from the registered param.